graphs/omen-thumbnailmapping/abis/Contract.json

```diff
@@ -0,0 +1,64 @@
+[
```

Comment:
Do you think there is a way we can re-use the ABI from the `src` folder? I don't love that we hardcode the ABI here, because every time we change the contract we need to also change it here.
One option that comes to mind is auto-generating the graph files (i.e. populating folder `generated`) using the `out` folder (created by `forge build`). What do you think?

Reply:
Yeah, I'd like that too! 

This file seems to only be used in `subgraph.yaml`, where we can do `../../out/...`.

So I removed this file and instead pointed it to `file: ./../../abi/omen_thumbnailmapping.abi.json`.

However, that doesn't change the fact that `omen_thumbnailmapping.abi.json` still needs to be updated manually.

But the issue I have is that `forge build` won't create ABI, it creates JSON with many other stuff, including ABI:

![Screenshot by Dropbox Capture](https://github.com/gnosis/labs-contracts/assets/15619339/6682484f-c128-44f1-8591-7b1776e22632)

So before, I extracted ABI manually and created that file `abi/omen_thumbnailmapping.abi.json`.

Do you have a suggestion about how this could be automated?

Reply:
Sure, you could expand the build step to something like
`build: "jq '.abi' root_path/out/OmenThumbnailMapping.sol/OmenThumbnailMapping.json > /abi/omen_thumbnail_abi.json
"`
Would that help?

Reply:
Ah yes! Some good outside-of-box thinking! 

Looks good, see the last commit 🙏